### ISSUE TITLE
[Feature] USO lora loading support

### ISSUE THREAD

**Issue by C00reNUT** [nunchaku-tech/nunchaku#650]
### Checklist

- [ ] 1. If the issue you raised is not a feature but a question, please raise a discussion at https://github.com/mit-han-lab/nunchaku/discussions/new/choose. Otherwise, it will be closed.
- [ ] 2. I will do my best to describe the issue in English.

### Motivation

Hello may I ask you would i be possible to include support for the USO model?

I shall be just a flux lora -

"We begin with FLUX.1 dev and the SigLIP pretrained model. For style alignment stage, we train on pairs for 23, 000 steps at batch size 16, learning rate 8e−5, resolution 768 and reward steps = 16, 000. For content-style disentanglement stage, we train on triplets for 21, 000 steps at batch size 64, learning rate 8e − 5, resolution 1024 and reward steps S = 18, 000. LoRA rank 128 is used throughout."

https://huggingface.co/bytedance-research/USO

**Comment by wencxxxxxx** [comment]
+1

**Comment by woct0rdho** [comment]
Hi @lmxyy , I'm wondering if ComfyUI-Nunchaku can be refactored in a similar way to Comfy-GGUF: Only override the model loader and the linear layer, so it's compatible with everything else in ComfyUI (LoRA/FBCache/memory offload...) and every model (Flux/Qwen-Image/Wan...) Then we can optimize the speed of LoRA later, and in many cases it's not the bottleneck.

**Comment by lmxyy** [comment]
> Hi [@lmxyy](https://github.com/lmxyy) , I'm wondering if ComfyUI-Nunchaku can be refactored in a similar way to Comfy-GGUF: Only override the model loader and the linear layer, so it's compatible with everything else in ComfyUI (LoRA/FBCache/memory offload...) and every model (Flux/Qwen-Image/Wan...) Then we can optimize the speed of LoRA later, and in many cases it's not the bottleneck.

Thanks for the advice. Since we’ve migrated to the Python backend, we’ll work on a more robust implementation to improve ComfyUI compatibility.
